README.md

```diff
@@ -1,7 +1,12 @@
 # espoo-dev
 
+Run project with docker  🐳
 
-# doc
+```bash
+docker-compose -f docker-compose.dev.yml up
```

Comment:
I think that is simpler if we use docker-compose.yml for dev and docker-compose.prod.yml for prod.
During daily usage have to add `-f docker-compose.dev.yml` to every command is not so coll, what do you think?